.github/workflows/ci.yml

```diff
@@ -17,6 +17,10 @@ jobs:
         run: cargo clippy --no-default-features --features="std log utils"
       - name: Run clippy with no_std
         run: cargo clippy --no-default-features --features="log"
+      - name: Run clippy for tokio feature
+        run: cargo clippy --features="tokio"
+      - name: Run clippy for async feature
+        run: cargo clippy --no-default-feature --features="async"
```

Comment:
It should be `--no-default-features` here. 😅

Reply:
I fixed, but there is a bigger problem. `async` feature can be only run with `+nightly` and I'm not sure that this version is available in workflows by default.

Reply:
It was fixed